data/network/src/commonMain/kotlin/social/androiddev/common/network/model/Application.kt

```diff
@@ -17,15 +17,24 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 data class Application(
-    // required attributes
     val id: String,
     val name: String,
+    @SerialName("vapid_key") val vapidKey: String,
+
+    // optional attributes
+    val website: String? = null,
+)
+
+@Serializable
+data class NewOauthApplication(
+    val id: String,
```

Comment:
probably good idea to break this out into it's own file with a javadoc on both of these models

Reply:
javadoc yes, but a separate file will only bloat it.
In the API Doc the application is re-used. So you'll probably first have a look into this file